app/views/publisher_information/what_managing_editors_can_do.html.erb

```diff
@@ -0,0 +1,13 @@
+<% content_for :browser_title, t("publisher_information.what_managing_editors_can_do.title") %>
+<div class="govuk-grid-row">
+  <%= render partial: "publisher_info_nav", locals: {current_page: managing_editors_path } %>
```

Comment:
We seem to be missing a space between the { and current_page. Looks like this is on the other ones too if you wouldn't mind?

Reply:
Added a space to all the views where this was missing

Reply:
⭐️